CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/Services/SignalRService.cs

```diff
@@ -80,7 +84,8 @@ private async Task ShowPopupAsync(string message)
                 await SecureStorage.SetAsync("popup_message", message);
                 await SecureStorage.SetAsync("popup_visible", "true");
```

Comment:
maybe it would be better to use local storage with some kind of SQLite? Rather than using Secure Storage? We could store all notifications there directly, Secure Storage is not designed for this I'm afraid. In Secure Storage we can hold token and other important information, but for larger ones maybe it's better to use InMemory storage such as SQLite

Reply:
I don't think there is any point in saving this data. If the user closed application without clicking on accept or rejected bottoms, all the same when he opens the application he will see the same popup page